build.gradle

```
@@ -72,6 +72,12 @@ dependencies {
 
     //redis
     implementation 'org.springframework.boot:spring-boot-starter-data-redis'
+
+    //Webflux
+    implementation 'org.springframework.boot:spring-boot-starter-webflux'
+
+    //thymeleaf
+    implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
```

```suggestion
    //redis
    implementation 'org.springframework.boot:spring-boot-starter-data-redis'

    //webflux
    implementation 'org.springframework.boot:spring-boot-starter-webflux'

    //thymeleaf
    implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
```
webflux만 첫 글자가 대문자로 되어있으니 조금 어색한 것 같습니다.

---

꼼꼼하십니다! 수정하겠습니다~!!